Rakefile

```diff
@@ -25,7 +25,7 @@ end
 namespace :admin do
   task :build do
     Dir.chdir('admin') do
-      sh 'ember build  --environment=production'
+      sh 'ember build production'
```

Comment:
will this require a new minimum version of `ember-cli`? if so, where/how will that be noted?

Reply:
@romulomachado can you me with the correct bits on this? I changed ^ that per our chat. Where should I list the version of ember-cli ? I see `admin/bower.json`, but I'm unsure if that's the right place.

Reply:
@jhirbour @alexdean This is more about `ember-cli-deploy` than `ember-cli`. 

Older versions of `ember-cli-deploy` won't work without the `--environment` flag, talkstar-frontend still uses an old version. 

Newer versions work fine with what's being changed here, donkey-frontend and programmer-frontend use a more recent version.